src/party_menu.c

```diff
@@ -5742,9 +5745,156 @@ static void Task_TryLearningNextMove(u8 taskId)
     }
 }
 
-static void CB2_ReturnToPartyMenuUsingRareCandy(void)
+u16 GetItemEVValue(const u8 *itemEffect)
 {
-    gItemUseCB = ItemUseCB_RareCandy;
+        // Assuming the EV value is stored at index [6] in the item effect array
+    return itemEffect[6];
+}
+
+void ItemUseCB_EVItem(u8 taskId, TaskFunc task)
```

Comment:
Could you explain the reason behind this function?

It looks like it would be much easier to replace `MAX_TOTAL_EVS` total in the medicine function with `GetCurrentEVCap()` Isn't this basically all you would need to do?

Also adding`B_EV_ITEMS_CAP' is not needed needed because `GetCurrentEVCap()` will give you the result needed. So if it is off it will be `MAX_TOTAL_EVS` if the config is off

Reply:
`B_EV_ITEMS_CAP` is for when you want to have EV caps but want to be able to use EV items to go above them, similar to how you can go over level caps with rare candies with that config.